src/app/ui/main_window.h

```diff
@@ -68,6 +68,7 @@ namespace app {
     void reloadMenus();
     void showNotification(INotificationDelegate* del);
     void showHome();
+    bool isSelectedHome();
```

Comment:
Hmm, I think I've to rename that function too. Just as a side note, there are several refactors that I should do but I've never time to do them. And sometimes, there are ways to do the same thing in two different ways (because a refactored-better way was created, but it didn't invalidate the old one). I've to write a document about that.